content/node/leanpocket.md

```diff
@@ -0,0 +1,94 @@
+---
+title: LeanPocket
+menuTitle: LeanPocket
+weight: 35
+description: Run multiple nodes on the same hardware without scaling your resource requirements.
+---
+
+LeanPocket (sometimes stylized as "LeanPOKT") is a configuration that allows multiple Pocket nodes running on a single server to consolidate their resources, resulting in better scalability and reduced hardware costs.
+
+LeanPocket is a feature available from version 0.9.2 and above.
+
+## Background
+
+Pocket Core is a single-threaded application. This means that if you wanted to run multiple Pocket nodes on the same server, each node, running its own isolated process, would each use their own resources: RAM, disk space, network throughput, etc. As many aspects of nodes use shared resources (copies of the blockchain, most notably) this was deemed very inefficient.
+
+For example, at the time of writing, a Pocket node requires a minimum of 500GB of disk space and 16GB of RAM, and that's not including the requirements of [the other chains the node may be servicing](/supported-blockchains).
+
+LeanPocket is an optimization to the core client to allow nodes to share resources, significantly reducing resource requirements. This is a non-consensus-breaking change to the client.
+
+<!-- Add charts here -->
+
+{{% notice style="info" %}}
+LeanPocket is specifically geared toward multiple nodes running on the same server. If you are running a single node, or have each of your nodes running on different server, LeanPocket will not have any benefit or effect.
+{{% /notice %}}
+
+## Setup
+
+To enable LeanPocket:
+
+1. Create a new `lean_nodes_keys.json` file inside your Pocket configuration directory (typically `.pocket`).
+
+2. Format the file with a JSON array of your node's private keys:
+
+   ```
+   [
+     { "priv_key": "<PRIVATE_KEY_1>" },
+     { "priv_key": "<PRIVATE_KEY_2>" }
+   ]
+   ```
+
+3. Set/update your validators:
+
+   ```
+   pocket accounts set-validators <path/to/lean_nodes_keys.json>
+   ```
+
+   {{% notice style="info" %}}
+   Note the plural form in the command. The option `set-validators` is distinct from `set-validator` and is only relevant for LeanPocket.
+   {{% /notice %}}
+
+4. Edit your node's `config.json` file to add `lean_pocket: "true";` in the `pocket_config` block.
```

Comment:
Depending on the user config files, lean_pocket may already be added and set to "false". We may want to include wording for them to search for lean_pocket before adding a block for it